sound/soc/sof/mediatek/mt8195/mt8195.c

```diff
@@ -316,6 +316,20 @@ static int mt8195_dsp_suspend(struct snd_sof_dev *sdev, u32 target_state)
 {
 	struct platform_device *pdev = container_of(sdev->dev, struct platform_device, dev);
 	int ret;
+	int reset_sw, dbg_pc, sleep_try = 2000, sleep_cnt = 0;
```

Comment:
initialized variables should be in separate line.
It does not make things easier to follow if you use variable initialized to 2000 than using it directly or even better, have a define for it in place.
int reset_sw, ret;
int sleep_cnt = 0;